test/nuts/convert/mdapi.nut.ts

```diff
@@ -90,6 +90,10 @@ describe('project convert mdapi NUTs', () => {
       expect(result.jsonOutput?.status).to.equal(0);
       expect(result.jsonOutput?.result).to.be.an('array').with.length.greaterThan(10);
       expect(fs.existsSync(convertedToSrcPath)).to.be.true;
+      expect(result.jsonOutput).to.not.be.undefined;
+      result.jsonOutput?.result.forEach((md) => {
+        expect(md.filePath.startsWith(convertedToSrcPath)).to.be.true;
```

Comment:
This NUT was testing that the output dir was created in L92 (which is true) but the metadata was still written to the first dir in the path.

![Screenshot 2024-07-15 at 7 29 28 PM](https://github.com/user-attachments/assets/146ebefe-fdaa-4c67-ae4e-a00539969df9)